### ISSUE TITLE
Very disappointed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#359]
```
I am a data analyst (among other things) who routinely works with large 
datasets sometimes requiring normalization.  My normal tools are MS Excel and 
Access.  So, when I heard about Refine I couldn't wait to try it.

I hate to criticize others' work so negatively, but what a disappointment!  
Here's a partial list of the issues I encountered.

. Refine could not "facet" a dataset of about 11K records on a column with a 
little more than 8K unique values ("too many values to display").  That's a 
fairly small dataset in the kind of work I do, and not a lot of unique values 
either.

. Could not filter on a simple NOT condition.  Apparently Refine filtering 
supports simple absolute string matching and for all other purposes uses 
"regular expressions" and/or other forms of code.  If I've misunderstood 
Refine's capabilities I apologize, but would like to point out that this is the 
extent of my understanding after scouring the available user documentation.

. Refine ran for 5 minutes faceting my 11K record dataset on a simple integer 
numeric field and never finished.  I finally shut Refine down as I had no idea 
how close to done it was.  A progress indicator would help a lot.

. No way to jump to a random ordinal record in the dataset.  Refine only lets 
you navigate to the beginning, end, next or previous record.  This is useless 
for datasets where one often needs to jump to the middle somewhere, especially 
in large datasets.

. Export does not export flags and stars!  I had to do two exports, one of only 
flagged records and the other of all records and rejoin in Access to work 
around this limitation.

. Export does not export in sorted sequence.  If you sort the dataset in Refine 
and then export it the sort sequence is lost in the output.

. And last but not least, please produce a real user's guide that gives 
explanations of all its main features and examples of their use as well as a 
proper table of contents and an index.  A function context help system (a la 
Microsoft, sorry to say) would also work.  Videos and scenarios such as the 
user documentation Web site provides are nice, but they're useless when one is 
trying to figure out how to do something different from the examples.

In short, I'm very disappointed because Refine's visualization features would 
be very useful in my work, but they are not available to me because of the 
product's severe limitations.  I'm sure it's a great product for users with 
really small datasets to manipulate, but that pretty much leaves me out.
```

Original issue reported on code.google.com by `jmangr...@gmail.com` on 12 Apr 2011 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
Thank you for your feedback. May I encourage you to use the mailing list 
instead of this issue tracking system? This is because you are not really 
filing a specific bug, or a specific feature request, but you're starting a 
discussion. The mailing list is appropriate for such a discussion.

The mailing list is also a place for you to ask how to do something. We will 
gladly answer your questions there.
```

Original comment by `dfhu...@gmail.com` on 12 Apr 2011 at 6:47

**Comment by GoogleCodeExporter** [comment]
```
dfhu...@gmail.com,

Thanks for steering me in the right direction.

As you can probably tell I'm not a veteran of OpenSource projects and not very 
familiar with the rules and customs of the community.  I apologize for that and 
will file my message in the mailing list, as you recommended.
```

Original comment by `jmangr...@gmail.com` on 14 Apr 2011 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Or conversely, if you've got a specific feature requests or problem reports, 
create a separate entry per problem/request so that they can be tracked 
appropriately.  

Providing the requested information (software versions, operating system, etc) 
will also help provide more useful responses.  The performance issues sound 
you've probably got your system underconfigured for what you're trying to do.

I'm going to close this.  Feel free to create new entries for specific problems 
-- or ask questions on the mailing list.
```

Original comment by `tfmorris` on 17 Apr 2011 at 4:10
- Changed state: **Invalid**
